atlp-rwanda/atlp-pulse-fn
# #DP-111 fix: Update profile picture

### What does this PR do?
This pull request allow the user to update their profile picture

### Description of the tasks to be completed
- [ ] A user should be able to change their avatar
 
### How should this be manually tested?
Use the heroku-review app linked to this pull request or click the link below:

https://devpulse-review-app-99.herokuapp.com/

To test on your local machine:

1. Make sure you clone the project (Backend and Frontend)
2. Checkout the respective branches for both the backend and frontend
3. Follow the steps below for the frontend

```js
git fetch
npm install
// create .env file and add the following line
// BACKEND_URL=<the link to your backend link>
npm run dev
open http://localhost:3000/
```
### Pre-requisites:
- Have both the backend and frontend projects

### Relevant trello story:
[#DP-111](https://trello.com/c/AYiItUdt/111-profile-picture-update)
